pkg/query/defaults.go

```diff
@@ -22,15 +23,33 @@ var (
 )
 
 // PermittedFormats stores all supported output formats
-var PermittedFormats = map[string]struct{}{
+var permittedFormats = map[string]struct{}{
 	"txt":  {},
 	"json": {},
 	"csv":  {},
 }
 
+func PermittedFormats() []string {
+	var aux []string
+	for format := range permittedFormats {
+		aux = append(aux, format)
+	}
+	sort.StringSlice(aux).Sort()
+	return aux
+}
+
 // PermittedSortBy sorts all permitted sorting orders
-var PermittedSortBy = map[string]results.SortOrder{
+var permittedSortBy = map[string]results.SortOrder{
 	"bytes":   results.SortTraffic,
 	"packets": results.SortPackets,
 	"time":    results.SortTime,
 }
+
+func PermittedSortBy() []string {
```

Comment:
Instead of creating the slice over and over, why not initialize it once (or, since it only has three items, just provide both slice and map to avoid the unnecessary code?